httpdocs/js/webots-cloud.js

```diff
@@ -161,14 +161,16 @@ document.addEventListener('DOMContentLoaded', function() {
       }
       const admin = project.email ? project.email.endsWith('@cyberbotics.com') : false;
       const type_name = (data.duration === 0) ? 'scene' : 'animation';
-      const url = data.url.startsWith('https://webots.cloud') ? document.location.origin + data.url.substring(20) : data.url
+      const url = data.url.startsWith('https://webots.cloud') ? document.location.origin + data.url.substring(20) : data.url;
+      const version_url = `https://github.com/cyberbotics/webots/tree/${data.version}`;
```

Comment:
I would rather point to the released package, so that people can download it right away:
```suggestion
      const version_url = `https://github.com/cyberbotics/webots/releases/tag/${data.version}`;
```